code/script.js

```diff
@@ -1,47 +1,219 @@
+// preventing the default behavior of a form which is refresh page on submit
+const handleSubmit = (e) => {
+  e.preventDefault()
+  
+  if (textInput.value === '') {
+    showMessage('Are you missing a name?', 'bot')
+    
+  } else if (qnr === 1) {  
+    userReply1()
+  } 
 }
 
-// Set up your eventlisteners here
+ // chatbot starts here
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000)
+  // Step 1
+const botReply1 = () => {         //declare function that is bots greeting
+  showMessage(`Hello there human, What's your name?`, 'bot')
+}
+
+const userReply1 = () => {        //declaring function that is users first message
+  const name = textInput.value    //storing user name for step 2
+  showMessage(name, 'user')       //the name is shown in users chat bubble
+  textInput.value = ""
+  qnr = qnr + 1
+
+  setTimeout(() => botReply2(name), 1000)   //trigger step 2
+}
+
+// Step 2
+const botReply2 = (name) => {     //declare function that is bots second message
+  showMessage(`Hello ${name}! Do you enjoy life on planet Earth? I heard it's a ticking time bomb.`, 'bot')
+
+  //input buttons with yes and no
+  inputWrapper.innerHTML = `      
+  <button id='yes'>Yes</button>
+  <button id='no'>No</button>
+  `
+  
+  //listen for "click" on buttons, and trigger the right user message depending on which button has been clicked
+  document.getElementById('yes').addEventListener('click', userReply2Yes)
+  document.getElementById('no').addEventListener('click', userReply2No) 
+}
+
+const userReply2Yes = () => {     
+  showMessage("Yes, I believe in humanity.", 'user') //message shown if user clicks "yes"  
+  let answer = "yes"
+  setTimeout(() => botReply3(answer), 1000)   //trigger step 3
+} 
+
+const userReply2No = () => {
+  showMessage("No, it's a miracle that we are still alive.", 'user')  //message shown if user clicks "no"
+  let answer = "no"
+  setTimeout(() => botReply3(answer), 1000)   //trigger step 3
+} 
+
+// Step 3
+const botReply3 = (answer) => {     
+  if (answer === "yes") {             //if user clicks yes, the following message is shown
+  showMessage(`Interesting, you seem so positive despite the pandemic. How will you solve it?`, 'bot')
+  
+  //input dropdown menu with 2 choices
+  inputWrapper.innerHTML = `
+  <select class="dropdown-menu" id="select">
+    <option value="" selected disabled>👇 Select an answer...</option>
+    <option id="quarantine" value="quarantine">Quarantine</option>
+    <option id ="flee" value="flee">FLEE!</option>
+  </select>
+  `
+
+} else if (answer === "no") {       //if user clicks no, the following message is shown
```

Comment:
Same goes with the reply3 function and the if/else ifs,, if there is only two possibilities else. But it doesn't really make a difference in terms of how the code runs, because it works really well as it is!